tests/tape/test_qnode.py

```diff
@@ -368,6 +368,44 @@ def func():
         # No differentiation required. No error raised.
         grad()
 
+    def test_unrecognized_keyword_arguments_validation(self):
+        """Tests that a UserWarning is raised when unrecognized keyword arguments are provided."""
+
+        # use two unrecognized methods, to confirm that multiple warnings are raised
+        unrecognized_one = "test_method_one"
+        unrecognized_two = "test_method_two"
+        warning_text = (
+            " is unrecognized, and will not be included in your computation. "
+            "Please review the QNode class or qnode decorator for the list of available "
+            "keyword variables."
+        )
+
+        expected_warnings = {
+            (UserWarning, unrecognized_one + warning_text),
+            (UserWarning, unrecognized_two + warning_text),

```

Comment:
If going with my suggestion above:
```suggestion
            (UserWarning, f"'{unrecognized_one}'{warning_text}"),
            (UserWarning, f"'{unrecognized_two}'{warning_text}"),
```